test/core/integration/fuzz/lockup-dynamic/createWithDurationsLD.t.sol

```diff
@@ -3,16 +3,9 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { ISablierLockup } from "src/core/interfaces/ISablierLockup.sol";
 import { Lockup, LockupDynamic } from "src/core/types/DataTypes.sol";
-import { Lockup_Integration_Shared_Test } from "./../../shared/lockup/Lockup.t.sol";
-
-contract CreateWithDurationsLD_Integration_Fuzz_Test is Lockup_Integration_Shared_Test {
-    uint256 internal streamId;
-
-    function setUp() public virtual override {
-        Lockup_Integration_Shared_Test.setUp();
-        streamId = lockup.nextStreamId();
-    }
+import { Integration_Test } from "./../../Integration.t.sol";
 
+contract CreateWithDurationsLD_Integration_Fuzz_Test is Integration_Test {
```

Comment:
me symmetry reason here, we should inherit from `Lockup_Dynamic_Integration_Fuzz_Test`